Referenced by issue #125448, (4.2) PyAPI: Assigning dicts as PropertyGroups no longer works in some cases
